pkg/api/schema/schema.proto

```diff
@@ -377,10 +377,11 @@ message VerifiableZAddRequest {
 }
 
 message TxRequest {
-	uint64      tx = 1;
-	EntriesSpec entriesSpec = 2;
-	uint64 sinceTx  = 3;
-	bool   noWait   = 4;
+	uint64      tx                = 1;
+	EntriesSpec entriesSpec       = 2;
+	uint64      sinceTx           = 3;
+	bool        noWait            = 4;
+	bool        keepReferencesUnresolved = 5;
```

Comment:
How about putting it in the `EntriesSpec` structure - for each type separately?

Reply:
though on that, it would require some additional changes on when the snapshot was requested